src/lib/Syntax.hs

```diff
@@ -1516,6 +1537,9 @@ builtinNames = M.fromList
   , ("Int64"   , TCExpr $ BaseType $ Scalar Int64Type)
   , ("Int32"   , TCExpr $ BaseType $ Scalar Int32Type)
   , ("Word8"   , TCExpr $ BaseType $ Scalar Word8Type)
+  , ("Int32Ptr", TCExpr $ BaseType $ ptrTy $ Scalar Int32Type)
+  , ("Word8Ptr", TCExpr $ BaseType $ ptrTy $ Scalar Word8Type)
+  , ("PtrPtr"  , TCExpr $ BaseType $ ptrTy $ ptrTy $ Scalar Word8Type)
```

Comment:
Why can't we just have a builtin that makes a pointer type?

Reply:
Yes, that would work too